### PR TITLE
Tweak - Section video background play once - show background fallback after video ends.

### PR DESCRIPTION
**Pull request by nuritsha**

### CODE REVIEW

File: assets/dev/js/frontend/handlers/section.js [hunk 1]
```diff
@@ -153,6 +159,7 @@ const BackgroundVideo = elementorModules.frontend.handlers.Base.extend( {
 			this.player.destroy();
 		} else {
 			this.elements.$backgroundVideoHosted.removeAttr( 'src' );
+			this.elements.$backgroundVideoHosted.off( 'ended' );
```
Comment (kobizz) [inline assets/dev/js/frontend/handlers/section.js]: You can just chain it

File: assets/dev/js/frontend/handlers/section.js [hunk 2]
```diff
@@ -111,7 +111,7 @@ const BackgroundVideo = elementorModules.frontend.handlers.Base.extend( {
 						case YT.PlayerState.ENDED:
 							self.player.seekTo( elementSettings.background_video_start || 0 );
 							if ( elementSettings.background_play_once ) {
-								self.player.stopVideo();
+								self.player.destroy();
```
Comment (kobizz) [inline assets/dev/js/frontend/handlers/section.js]: Why?

File: assets/dev/js/frontend/handlers/section.js (hunk 1 above)
Comment (nuritsha) [reply assets/dev/js/frontend/handlers/section.js]: done

File: assets/dev/js/frontend/handlers/section.js (hunk 2 above)
Comment (nuritsha) [reply assets/dev/js/frontend/handlers/section.js]: in order to remove the iframe completely

**Pull request merged**